src/elements/components/Spinner.tsx

```diff
@@ -6,7 +6,7 @@ function FeatherySpinner() {
     <Spinner
       animation='border'
       style={{
-        color: 'white',
+        color: '#bdc2ce', // gray-50
```

Comment:
the <form> html native element hasn't rendered by the time this has rendered for the auth loading...I tried a few things but ended up just making it grey so it is visible in most situations

Reply:
how does that look visually? if you need UI suggestions gabriela could probably help here too

Reply:
<img width="1475" alt="image" src="https://user-images.githubusercontent.com/923485/197865457-2c6355fc-4ad4-47b5-8b75-8aa8b55209ee.png">

I talked to Eric a bit about it, and doesn't seem like there's a reliable way to get the background color under the spinner without needing to traverse up the Cells to find which cell (if any) is responsible for the visible background color

Also the `<form>`* html native element hasn't rendered before the spinner is used while waiting on auth, so that would also require some additional work because the spinner is set in the mount/unmount useEffect

* what I was trying to say in my first message - didn't realize github parsed the form tag I had written out

Reply:
another way is you could do like a black border and white background so it's visible everywhere, but not sure if that looks good